lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -175,9 +174,11 @@ export default class BitcoinProcessor {
       }
     }
 
-    console.info(`Returning transactions since ${since ? 'block ' + TransactionNumber.getBlockNumber(since) : 'begining'}...`);
-    let transactions = await this.transactionStore.getTransactionsLaterThan(since, this.pageSize);
```

Comment:
Design: What do you think if we remove `getTransactionLaterThan()` method completely from the interface?

Reply:
Yes I like that. Right now only SpendingMonitor uses it, and it is grabbing by blocks too. I will create a new issue and separate pr for it.